### DISCUSSION TITLE
Error on installation

### DISCUSSION THREAD

**Discussion by united-vision** [Laravel-Backpack/community-forum#294] [Q&A (Help)] [answered]
Root package 'backpack/crud' cannot require itself in its composer.json

**Comment by pxpm** [comment, marked as answer]
Hello @united-vision 

I cannot see where we do require it in composer: https://github.com/Laravel-Backpack/CRUD/blob/master/composer.json

Can you elaborate  a little bit on your problem so we can try to help ?

Thanks,
Pedro
